scripts/report-libcall.py

```diff
@@ -0,0 +1,38 @@
+#
+# report-libcall.py
+#
+# uftrace-option: --nest-libcall -F .*@plt
+#
+
+import os
+import operator
+
+libcall_map = {}
+
+def uftrace_begin():
+    pass
+
+def uftrace_entry(ctx):
+    _name = ctx["name"]
+    if libcall_map.has_key(_name):
+        libcall_map[_name] += 1
+    else:
+        libcall_map[_name] = 1
+
+def uftrace_exit(ctx):
+    pass
+
+def uftrace_end():
+    global libcall_map
+    sorted_dict = sorted(libcall_map.items(), key=operator.itemgetter(1), reverse=True)
+
+    pid = os.getpid()
+    with open("/proc/%s/comm" % pid) as proc_comm:
```

Comment:
I think if there's a way to identify if the script is execution at record or replay time.  This code only makes sense when it runs at record time.